ironfish-cli/src/commands/deposit.ts

```diff
@@ -76,17 +76,17 @@ export default class Bank extends IronfishCommand {
       this.exit(1)
     }
 
-    const { canSend, errorReason } = await this.verifyCanSend(flags)
+    const graffiti = (await this.client.getConfig({ name: 'blockGraffiti' })).content
+      .blockGraffiti
+    Assert.isNotUndefined(graffiti)
```

Comment:
Small update here. We're checking blockGraffiti twice in the code. Removing this redundancy.